tests/unit/test_fvm_multi.cpp

```diff
@@ -328,6 +329,337 @@ TEST(fvm_multi, mechanism_indexes)
     }
 }
 
+namespace {
+    double wm_impl(double wa, double xa) {
```

Comment:
what is `wm`?

Reply:
Weighted mean; when it moved above the test body, it lost its context. Will add comment.